react-client/src/pages/plan/overview/overview.js

```diff
@@ -125,67 +139,73 @@ function PlanOverview() {
 
             <div className="plan-steps-step">
               <p>
-                <Link to={`/dashboard/dmp/${dmp.id}/funders`}>Funders</Link>
+                <Link id="step-project-funders" to={`/dashboard/dmp/${dmp.id}/funders`}>Funders</Link>
               </p>
-              <div className={"step-status status-" + dmp.stepStatus.funders[0]}>
+              <div
+                id="step-project-funders-status"
+                aria-describedby="step-project-funders step-project-funders-status"
+                className={"step-status status-" + dmp.stepStatus.funders[0]}>
                 {dmp.stepStatus.funders[1]}
               </div>
             </div>
 
             <div className="plan-steps-step">
               <p>
-                <Link to={`/dashboard/dmp/${dmp.id}/project-details`}>
+                <Link id="step-project-details"
+                  to={`/dashboard/dmp/${dmp.id}/project-details`}>
                   Project Details
                 </Link>
               </p>
 
-              <div className={"step-status status-" + dmp.stepStatus.project[0]}>
+              <div
+                id="step-project-details-status"
+                aria-describedby="step-project-details step-project-details-status"
+                className={"step-status status-" + dmp.stepStatus.project[0]}>
                 {dmp.stepStatus.project[1]}
               </div>
-              {dmp.errors.get("project") && (
-                <div className={"step-status status-error"}>
-                  Review Needed
-                </div>
-              )}
+
             </div>
 
             <div className="plan-steps-step">
               <p>
-                <Link to={`/dashboard/dmp/${dmp.id}/contributors`}>
+                <Link id="step-project-contributors" to={`/dashboard/dmp/${dmp.id}/contributors`}>
                   Contributors
                 </Link>
               </p>
 
-              <div className={"step-status status-" + dmp.stepStatus.contributors[0]}>
+              <div
+                id="step-project-contributors-status"
+                aria-describedby="step-project-contributors step-project-contributors-status"
+                className={"step-status status-" + dmp.stepStatus.contributors[0]}>
                 {dmp.stepStatus.contributors[1]}
               </div>
-
-              {dmp.errors.get("contributors") && (
```

Comment:
Why was this removed? Didn't we decide on this to show that action is required on this if there are errors?

Reply:
This was originally removed for accessibility reasons. However, we decided to keep it for now since we have a global message on the top. This label will notify where the attention is needed.